lightning_client.go

```diff
@@ -1436,6 +1442,42 @@ func (s *lightningClient) ChannelBackups(ctx context.Context) ([]byte, error) {
 	return resp.MultiChanBackup.MultiChanBackup, nil
 }
 
+func (s *lightningClient) SubscribeChannelBackups(ctx context.Context) (
```

Comment:
nit: Should have a godoc comment as well. Can just copy from the interface above.